### PR TITLE
🦄 [ui] wire in select action

### PR DESCRIPTION
**Pull request by clanam**
Screen Shot 2019-08-15 at 4 55 03 PM

### CODE REVIEW

File: emoji-ui/src/components/EmojiItem.js [hunk 1]
```diff
@@ -9,18 +9,36 @@ class EmojiItem extends Component {
...
+  }
+
   render() {
-    const { clickHandler } = this.props;
-    const { isSelected, url, name } = this.props.emoji;
+    const { isSelected, name, url } = this.props.emoji;
     const styles = { backgroundImage: `url(${url})` };
-    const containerClasses = `emoji-container${isSelected ? ' emoji-container--selected' : ''}`;
+    let containerClasses = 'emoji-container';
+    if (isSelected) {
```
Comment (clanam) [inline emoji-ui/src/components/EmojiItem.js]: i might have gotten a bit carried away... 😓

File: emoji-ui/src/components/SourceList.js [hunk 2]
```diff
@@ -26,9 +27,9 @@ class SourceListBase extends Component {
   }
 
   render() {
-    const { sourceEmoji } = this.props;
+    const { sourceEmoji, sourceEmojiActions } = this.props;
     const firstBatch = sourceEmoji.slice(0, 200);
-    return <EmojiList sourceEmoji={firstBatch} />;
+    return <EmojiList sourceEmojiActions={sourceEmojiActions} sourceEmoji={firstBatch} />;
```
Comment (clanam) [inline emoji-ui/src/components/SourceList.js]: this is why i think sourceList and emojiList should merge -- just do pagination in selector and boom! no more need for this wrapping!

File: emoji-ui/src/components/EmojiItem.js [hunk 3]
```diff
@@ -9,18 +9,36 @@ class EmojiItem extends Component {
...
+      containerClasses += ` ${containerClasses}--selected`;
+    }
+    const ariaLabel = `Emoji ${name}`;
 
     return (
       <div className={containerClasses}>
-        <button style={styles} className='emoji-item' onClick={() => clickHandler(name)}></button>
+        <button
+          alia-label={ariaLabel}
```
Comment (lisatsewang) [inline emoji-ui/src/components/EmojiItem.js]: nice!

Comment (gmkohler) [reply emoji-ui/src/components/EmojiItem.js]: should it be `aria-label` instead of `alia-label`

File: emoji-ui/src/components/SourceList.js (hunk 2 above)
Comment (lisatsewang) [reply emoji-ui/src/components/SourceList.js]: sounds good! will be working toward removing the SourceList wrapper.

**Comment by lisatsewang** [comment]
i am going to merge this PR since i am going to work on top of this PR.

**Pull request merged**